component-template/{{ cookiecutter.slug }}/Makefile

```
@@ -0,0 +1,41 @@
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := all
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+DOCKER_CMD   ?= docker
+DOCKER_ARGS  ?= run -v "$${PWD}:/component" --workdir /component
+
+JSONNET_FILES   ?= $(shell find . -type f -name '*.*jsonnet' -or -name '*.libsonnet')
+JSONNETFMT_ARGS ?= --in-place
+JSONNET_IMAGE   ?= docker.io/bitnami/jsonnet:latest
+JSONNET_DOCKER  ?= $(DOCKER_CMD) $(DOCKER_ARGS) --entrypoint=jsonnetfmt $(JSONNET_IMAGE)
+
+YAML_FILES      ?= $(shell find . -type f -name '*.yaml' -or -name '*.yml')
+YAMLLINT_ARGS   ?= --no-warnings
+YAMLLINT_CONFIG ?= .yamllint.yml
+YAMLLINT_IMAGE  ?= docker.io/cytopia/yamllint:latest
+YAMLLINT_DOCKER ?= $(DOCKER_CMD) $(DOCKER_ARGS) $(YAMLLINT_IMAGE)
+
+.PHONY: all
+all: lint
+
+.PHONY: lint
+lint: lint_jsonnet lint_yaml
+
+.PHONY: lint_jsonnet
+lint_jsonnet: $(JSONNET_FILES)
+	$(JSONNET_DOCKER) $(JSONNETFMT_ARGS) --test -- $?
+
+.PHONY: lint_yaml
+lint_yaml: $(YAML_FILES)
+	$(YAMLLINT_DOCKER) -f parsable -c $(YAMLLINT_CONFIG) $(YAMLLINT_ARGS) -- $?
+
+.PHONY: format
+format: format_jsonnet
+
+.PHONY: format_jsonnet
+format_jsonnet: $(JSONNET_FILES)
+	$(JSONNET_DOCKER) -- chmod +x $$(command -v jsonnetfmt) && jsonnetfmt $(JSONNETFMT_ARGS) -- $?
```

Why do we need the `chmod +x` here?

---

Oh, that is a leftover.

```suggestion
	$(JSONNET_DOCKER) $(JSONNETFMT_ARGS) -- $?
```